tests/urlpatterns_reverse/tests.py

```diff
@@ -574,6 +574,25 @@ def test_namespaces_with_variables(self):
         self.assertEqual('/inc70/', reverse('inc-ns5:inner-nothing', args=['70']))
         self.assertEqual('/inc78/extra/foobar/', reverse('inc-ns5:inner-extra', args=['78', 'foobar']))
 
+    def test_nested_app_lookup(self):
+        "A nested current_app should be split in individual namespaces: see #24904"
+        self.assertEqual('/ns-included1/test4/inner/', reverse('inc-ns1:testapp:urlobject-view'))
+        self.assertEqual('/ns-included1/test4/inner/37/42/', reverse('inc-ns1:testapp:urlobject-view', args=[37, 42]))
```

Comment:
Are the coding style guidelines preferred over local consistency? I actually prefer the vertical alignment with the url on the left and reverse() on the right, which makes it much more comprehensible in my opinion. I can still make that fit the 119 character limit.